data/strings/strings.txt

```diff
@@ -25519,6 +25519,142 @@
     zh-Hans = 摄像头信息
     zh-Hant = 超速警告
 
+  [carplay_continue_on_the_phone]
+    comment = Displayed on the phone screen. Button to display maps on the phone screen instead of a car
+    tags = ios
+    en = Tap here to use app on the phone
```

Comment:
There are already strings for Android Auto, can we reuse them (and maybe update texts/ids to fit better both scenarios)?

```
	<!-- Text on the Android Auto placeholder screen that maps are displayed on the phone screen -->
	<string name="aa_used_on_the_phone_screen">You are now using Organic Maps on the phone screen</string>
	<!-- Text on the phone placeholder screen that maps are displayed on the car screen -->
	<string name="aa_used_on_the_car_screen">You are now using Organic Maps on the car screen</string>
	<!-- Displayed on the phone screen. Android Auto connected -->
	<string name="aa_connected_title">You are connected to Android Auto</string>
	<!-- Displayed on the phone screen. Button to display maps on the phone screen instead of a car -->
	<string name="aa_continue_on_the_phone">Continue on the phone</string>
	<!-- Displayed on the Android Auto screen. Button to display maps on the car screen instead of a phone. Must be no more than 18 symbols! -->
	<string name="aa_continue_in_the_car">To the car screen</string>
```

Reply:
Yes, I think we can reuse strings displayed on car head unit. We could also reuse strings displayed on the phone, but probably after small redesign (to align iOS UI with Android: text description + button). I will look at this.

Reply:
Reused existing strings from Android Auto: `aa_used_on_the_phone_screen`, `aa_used_on_the_car_screen`, `aa_continue_on_the_phone` and `aa_continue_in_the_car`. Should we rename them (for example use `car_` prefix instead of `aa_` ?)? If yes, maybe do it in separate PR?

Reply:
I like `car_` and I would do it in this PR because you already changed the comments.

Reply:
Renamed to `car_`